css/pop-up.css

```diff
@@ -0,0 +1,107 @@
+.pop-up {
+  scrollbar-gutter: stable;
+  padding: 4rem 1.6rem;
+  display: flex;
+  flex-direction: column;
+  gap: 1.6rem;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  width: 90%;
```

Comment:
Great work on your popup for the mobile version! **For the desktop version, there are a couple of suggestions I'd like to make**:

- [ ] The size of the popup seems a **bit too big on desktop**. You could adjust it by removing or changing the width from **90%** to match the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-468&t=Lmbe0KVCHoQwSAR4-0). 🔍
- [ ] The space between the `X` and the top of the popup looks a bit larger than in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-468&t=Lmbe0KVCHoQwSAR4-0). To give your project a **more polished and professional look**, you could consider adjusting it to match the template. 😉

  ### Your design:
  ![image](https://github.com/M-Anwar-Hussaini/Portfolio/assets/107952724/b6ad3f18-3d3b-4167-a30e-f296674d9246)

  ### Figma design:
  ![image](https://github.com/M-Anwar-Hussaini/Portfolio/assets/107952724/a77ca01f-e05b-4fca-a0f0-ec09e02c58d3)